node-deb

```diff
@@ -730,6 +747,8 @@ replace_vars() {
   declare -r target_file="$2"
   declare -r permissions="$3"
 
+  lf=$'\n'
```

Comment:
Why is `lf` being set here? Can you add a comment please.

Reply:
sed on macOS does not properly interpret `\n` in the replace pattern so we either need to do a literal line break (ugly) or assign `\n` to a var and do it that way.